src/commands/migrate-service.ts

```diff
@@ -0,0 +1,271 @@
+import Command from "@oclif/command";
```

Comment:
it looks like this file migrates all the services so it should be renamed to migrate-services.ts